src/VS.Web.CG.Mvc/Common/RequiredFileEntity.cs

```diff
@@ -23,6 +24,15 @@ public RequiredFileEntity(string outputPath, string templateName)
             OutputPath = outputPath;
         }
         
+        public RequiredFileEntity(string outputPath, string templateName, List<string> altPaths)
+            : this(outputPath, templateName)
+        {
+            if (altPaths != null)
```

Comment:
Minor: Instead of those this check and the default value assignment, you can just have:
```csharp
    AltPaths = altPaths ?? new List<string>();
```